main.go

```diff
@@ -156,6 +156,8 @@ func main() {
 	router.Handle("/cookies{uri:.*}", createReverseProxy("cookies", cookiesControllerURL))
 
 	router.Handle("/datasets/{uri:.*}", createReverseProxy("datasets", datasetControllerURL))
+	router.Handle("/economy/inflationandpriceindices/datasets/understandingthedifferentapproachesofmeasuringowneroccupiershousingcosts/current", createReverseProxy("/economy/inflationandpriceindices/datasets/understandingthedifferentapproachesofmeasuringowneroccupiershousingcosts/current", datasetControllerURL))
```

Comment:
@mr-nick17 

So am I right in thinking this is the "hack" everyone is talking about where the new template that was made is only going to work for this particular url/dataset?

Reply:
Yes this will go away when the routing pr is merged